book/10-git-internals/sections/environment.asc

```diff
@@ -1,138 +1,138 @@
 
-*`GIT_CEILING_DIRECTORIES`* controls the behavior of searching for a `.git` directory.
-If you access directories that are slow to load (such as those on a tape drive, or across a slow network connection), you may want to have Git stop trying earlier than it might otherwise, especially if Git is invoked when building your shell prompt.
+*`GIT_CEILING_DIRECTORIES`* steuert das Suchverhalten nach einem `.git`-Verzeichnis.
+Wenn Sie auf Verzeichnisse zugreifen, die nur langsam geladen werden können (z.B. auf einem Bandlaufwerk oder über eine langsame Netzwerkverbindung), möchten Sie möglicherweise, dass Git den Versuch vorzeitig abbricht, insbesondere wenn Git in der Kommandozeile aufgerufen wird.
 
-*`GIT_WORK_TREE`* is the location of the root of the working directory for a non-bare repository.
-If `--git-dir` or `GIT_DIR` is specified but none of `--work-tree`, `GIT_WORK_TREE` or `core.worktree` is specified, the current working directory is regarded as the top level of your working tree.
+*`GIT_WORK_TREE`* ist der Speicherort des Stammverzeichnisses eines Arbeitsverzeichnisses für ein non-bare Repository.
+Wenn `--git-dir` oder `GIT_DIR` angegeben ist, jedoch nicht `--work-tree`, `GIT_WORK_TREE` oder `core.worktree`, wird das aktuelle Arbeitsverzeichnis als oberste Ebene Ihrer Arbeitsbaums betrachtet.
 
-*`GIT_INDEX_FILE`* is the path to the index file (non-bare repositories only).
+*`GIT_INDEX_FILE`* ist der Pfad zur Indexdatei (nur für non-bare Repositories).
 
-*`GIT_OBJECT_DIRECTORY`* can be used to specify the location of the directory that usually resides at `.git/objects`.
+*`GIT_OBJECT_DIRECTORY`* kann verwendet werden, um den Speicherort des Verzeichnisses anzugeben, das sich normalerweise in `.git/objects` befindet.
 
-*`GIT_ALTERNATE_OBJECT_DIRECTORIES`* is a colon-separated list (formatted like `/dir/one:/dir/two:…`) which tells Git where to check for objects if they aren't in `GIT_OBJECT_DIRECTORY`.
-If you happen to have a lot of projects with large files that have the exact same contents, this can be used to avoid storing too many copies of them.
+*`GIT_ALTERNATE_OBJECT_DIRECTORIES`* ist eine durch Doppelpunkte getrennte Liste (formatiert wie `/dir/one:/dir/two:… `), die Git mitteilt, wo nach Objekten gesucht werden soll, wenn sie sich nicht in` GIT_OBJECT_DIRECTORY` befinden.
+Wenn Sie viele Projekte mit großen Dateien haben, die genau den gleichen Inhalt haben, können Sie damit vermeiden, dass zu viele Kopien davon gespeichert werden.
 
 
-==== Pathspecs
+==== Pfadspezifikation (engl. Pathspec)
 
-A ``pathspec'' refers to how you specify paths to things in Git, including the use of wildcards.
-These are used in the `.gitignore` file, but also on the command-line (`git add *.c`).
+„Pathspec“ bezieht sich darauf, wie Sie Pfade in Git angeben, einschließlich der Verwendung von Platzhaltern.
```

Comment:
Die Anführungsstriche unterscheiden sich vom englischen Original. Ist das in Ordnung?

Reply:
Die Frage gilt auch für die Anführungsstriche im restlichen Text, da kommt das noch häufiger vor

Reply:
Richtig! Es sollen die "deutschen", typografisch richtigen Anführungszeichen verwendet werden.
Schau dir die Datei [Special_Characters.asc](https://github.com/progit/progit2-de/blob/master/Special_Characters.asc) an.

Reply:
Alles klar, danke!